### DISCUSSION TITLE
Great Stuff

### DISCUSSION THREAD

**Discussion by sivanmu** [jamesdh/kompromat#2] [Ideas]
I was wondering, if this tool could also leave different war pictures in the reviews? But, I suppose, google will not let it stay there for a long time. Especially, if these pictures/ will contain injured people.

**Comment by jamesdh** [comment]
See https://github.com/jamesdh/kompromat#modifying-the-review-text-and-photo

You can replace the message/photo with whatever you choose!

**Reply by sivanmu** [reply]
Hey James, right, I have seen it. I meant, if there was a possibility to insert multiple pictures. But it is just an idea. 
Thanks for doing this, I will try it out today. 
Cheers, 
Mariia

**Reply by jamesdh** [reply]
I'll see if I can add that today, to allow multiple messages + photos, and then randomly pick one for each review.
